float64[] jointPositions
---
arm_control/Pose solution
int32 result
